settings.gradle

```diff
@@ -1,3 +1,20 @@
+pluginManagement {
+    repositories {
+        google()
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
+
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
```

Comment:
Reading the docs this line means the gradle sync will fail is there are repositories outside of setting.gradle. Looks ok to me.